docker/docker-jaxrs-dockerfile/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM java:jre-alpine
```

Comment:
It should be openJRE. The image is listed here https://hub.docker.com/_/java/ . Just did a clean build and it works for me 

https://github.com/docker-library/openjdk/blob/54c64cf47d2b705418feb68b811419a223c5a040/8-jre/alpine/Dockerfile